### PR TITLE
feat: search using filters and global search

### PR DESCRIPTION
**Pull request by shobit1337**
- User can Search in global search for games
- Show more will redirect to browse page
- User can also set search in filters.

### CODE REVIEW

**Review by someshtiwarii — APPROVED** [review]
Clean code! Looks good to me .

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -11,10 +12,7 @@ const Navbar = () => {
   const activeNav = { fontWeight: 600, color: 'var(--light-color)' };
```
Comment (ScoobyDubeyDo) [inline src/components/Navbar/Navbar.jsx:12]: **suggestion**: Avoid inline stylings

File: src/components/Search/components/SearchResults/SearchResults.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { searchItems } from '../../../../utils/filters';
+import { getAllProducts } from '../../../../utils/products';
```
Comment (ScoobyDubeyDo) [inline src/components/Search/components/SearchResults/SearchResults.jsx:3]: **suggestion**: Use barrel export

**Review by anjali1102 — APPROVED** [review]

**Review by ScoobyDubeyDo — APPROVED** [review]
Good Naming conventions used
Liked the folder structure
Gave a few suggestions, try implementing them if possible

**Pull request merged**